graphs/DataVisualisation/SingleNumber/index.js

```diff
@@ -31,12 +34,15 @@ module.exports = React.createClass({
   },
 
   render: function() {
-    var loading = this.props.loading,
+    var loading    = this.props.loading,
+        emptyState = this.props.emptyState,
         className;
-
     if (loading) {
```

Comment:
You could make this neater with the classname module.
`var classnames = require('classnames');`
```js
var classes = classnames({
      "hui-SingleNumber--loading": loading,
      "hui-SingleNumber--emptyState": emptyState
    }, "hui-SingleNumber");
```